### ISSUE TITLE
Voicechat Stuttering/Choppy

### ISSUE THREAD

**Issue by spankyyy** [Facepunch/garrysmod-issues#5178]
Protocol version 24
Exe version 2021.12.15 (garrysmod)
Exe build: 21:16:15 Feb  7 2022 (8482) (4000)
GMod version 2022.02.07, branch: x86-64, multicore: 1
Windows 64bit

Voicechat starts to get Choppy/Stutter after a couple minutes of playtime.
This has been an issue ever since the introduction of a higher quality voicechat.
Many of my friends have reported to have the same issue
We all tried verifying our game caches but it didn't work
"snd_restart" fixes the issue temporarily 
This has been an issue for 64bit and if I recall correctly it was also problematic for 32bit  

How to reproduce:
There is not actual way to reproduce but being in a crowded voicechat or having lower fps makes the bug appear more often
and even sometimes a couple seconds after doing snd_restart

Before:
https://user-images.githubusercontent.com/75406797/157161723-e154544f-07f9-40d4-974b-e7f88a46b708.mp4

After:
https://user-images.githubusercontent.com/75406797/157161797-d50e87e5-f1d2-4364-8f4a-f537122d802a.mp4

**Comment by Fasteroid** [comment]
This has plagued multiple people on my server, myself included.

**Comment by FlorianLeChat** [comment]
This issue is probably also related to https://github.com/Facepunch/garrysmod-issues/issues/5008.

**Comment by MyDimeIsUp** [comment]
Surprised they've let this issue exist as long as it does. Completely ruins the experience. This has been what... almost a year?

**Comment by MyDimeIsUp** [comment]
bumping

**Comment by Fasteroid** [comment]
In my case both client and server are 64 bit.
________________________________

**Comment by wrefgtzweve** [comment]
Yeah I feel like that plays into it, it's hard to test but *maybe* 64bit servers cause it or increase its frequency.

**Comment by Fasteroid** [comment]
It happens on 64 bit server with 32 bit client as well.  Just observed while working on web development in awesomium.
